ke/__init__.py

```diff
@@ -54,7 +56,9 @@
 def re(pattern: AnyStr, flavor: Optional[Flavor] = None) -> AnyStr:
     # TODO: LRU cache
     if _is_bytes_like(pattern):
+        asm.InlineFlag.PATTERN_IS_BYTES_LIKE = True
```

Comment:
This might be better placed in some global values-per-compilation dictionary somewhere.